packages/flutter_tools/test/devfs_test.dart

```diff
@@ -359,8 +359,14 @@ class MockVMService extends BasicMock implements VMService {
   VM get vm => _vm;
 
   Future<Null> setUp() async {
-    _server = await HttpServer.bind(InternetAddress.LOOPBACK_IP_V6, 0);
-    _httpAddress = Uri.parse('http://[::1]:${_server.port}');
+    try {
+      _server = await HttpServer.bind(InternetAddress.LOOPBACK_IP_V6, 0);
+      _httpAddress = Uri.parse('http://[::1]:${_server.port}');
+    } on SocketException {
```

Comment:
SocketException could happen for other reasons. Is there a way to narrow it down more?